src/styles/layout.css

```diff
@@ -469,12 +469,12 @@ iframe {
 hr {
```

Comment:
We should add these as global styles in Theme.js - currently these elements aren't visible on dark mode.

Reply:
shall we create a new ticket or do it in this pr?